test/issuehunter.js

```diff
@@ -491,6 +491,39 @@ contract('Issuehunter', function (accounts) {
       })
     })
 
+    context('fund overflow', function () {
+      const issueId = nextCampaignId()
+      const funder = sampleAccount()
+      const txValue1 = (2 ** 32) - 1
```

Comment:
Great! :thumbup:

Did you check that it fails without the requirement?

Reply:
Actually it fails also without the `require`

Reply:
So, do you prefer that I only add the tests and remove the `require` in the contract?

Reply:
How is it possible that it fails without the require? What is the test output?